spec/conta_azul_api/http_response_spec.rb

```diff
@@ -0,0 +1,63 @@
+RSpec.describe ContaAzulApi::HttpResponse do
+  let(:product_payload) { File.read('spec/fixtures/products_endpoints/find_by_id.json') }
```

Comment:
WDYT about follow the approach `Setup -> Exercise -> Validation` on `it` blocks instead of use lets and subjects?

Reply:
Sure man 😄 

I changed that in [bfbda67](https://github.com/paulo-silva/conta_azul_api/pull/14/commits/bfbda67a91cbf0d49b05fbab077fbc06502a9b8e)